test/manipulation.test.js

```diff
@@ -57,6 +57,20 @@ describe('manipulation', function () {
       });
     });
 
+    it('should create instance (promise variant)', function (done) {
+      Person.create({name: 'Anatoliy'})
+      .then (function (p) {
```

Comment:
This will be always subjective. The rule I prefer is that whenever a statement is spread on more than one line, all subsequent lines are indented by two spaces, i.e. "just indent the first .then". This is also the rule we apply in supertest-based tests:

```js
supertest(app).get('/')
  .set(...)
  .send(..)
  .expect(...)
  .end(done);
```

> The space after function is not... if there is no space, it looks more like it is calling a function
rather than defining it (my editor actually highlights them differently).

Again, there isn't an objective way how to decide this. We are following the [Google coding style](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Function_Declarations_Within_Blocks#Function_Declarations_Within_Blocks) (which is what Node follows too AFAIK) where the anonymous functions are always declared as `function(...args...)`, no space after the `function` keyword.

> I notice that *most* other places in the file, the callbacks are defined with a space after function 

Yes, we were not consistent in keeping the coding style consistent. We had the same problem in strongloop/loopback, where we eventually decided to add `jscs` as a pretest script and fix all style violations manually.